docs/usecases/bids-fmriprep-workflow-NP.sh

```diff
@@ -0,0 +1,165 @@
+cd ..
+
+#
+# Install dataset to be analyzed (no data - analysis might run in the cloud or on HPC)
+#
+# In original paper name for the dataset was used as is, and placed at the
+# top level.  Here, to make this demo easier to apply to other studies,
+# and also check on other datasets, we install input dataset under a generic
+# "data/bids" path.  "data/" will also collect all other derivatives etc
+mkdir data
+
+# For now we will work with minimized version with only 2 subjects
+# datalad install -d . -s ///openneuro/ds000003 data/bids
+datalad install -d . -s https://github.com/ReproNim/ds000003-demo data/bids
+
+
+#
+# Licenses
+#
+
+# we will not prepopulate this one
+mkdir licenses/
+echo freesurfer.txt > licenses/.gitignore
+
+cat > licenses/README.md <<EOF
+
+Freesurfer
+----------
+
+Place your FreeSurfer license into freesurfer.txt file in this directory.
+Visit https://surfer.nmr.mgh.harvard.edu/registration.html to obtain one if
+you don't have it yet - it is free.
+
+EOF
+datalad save -m "DOC: licenses/ directory stub" licenses/
+
+
+#
+# Execution.
+#
+# That is where access to the powerful resource (HPC) etc would be useful.
+# Every of those containerized apps might need custom options to be added.
+#
+#
+
+# Define common parameters for the reproman run
+
+# ReproMan orchestrator to be used - determines how data/results would be
+# transferred and execution protocoled
+# Use  reproman run --list orchestrators  to get an updated list
+RM_ORC=datalad-pair-run  # ,plain,datalad-pair,datalad-local-run
+
+# Which batch processing system supported by ReproMan will be used
+# Use  reproman run --list submitters  to get an updated list
+# RM_SUB=condor,pbs,local
+
+# Which resource to use
+# It would require (if was not done before) to configure
+# a resource where execution will happen.  For now will just use smaug below.
+# TODO: provide pointers to doc ( ;-) )
+# RM_RESOURCE=
+
+#RM_RESOURCE=discovery
+#RM_SUB=PBS
+#
+# Necessary modules to be loaded in that session:
+#  - singularity/2.4.2
+# Necessary installations/upgrades to be done (TODO: contact John)
+#  - datalad (0.11.6, TODO: release first)
+#  - datalad-container
+
+RM_RESOURCE=smaug
+RM_SUB=condor
+
+# TODO: at reproman level allow to specify ORC and SUB for a resource, so there would
+#   be no need to specify for each invocation.  Could be a new (meta) resource such as
+#   "smaug-condor" which would link smaug physical resource with those parameters
+# TODO: point to the issue in ReproMan
+
+# 1. bids-mriqc -- QA
+
+# Q/TODO: Is there a way to execute/reference the container?
+#   for now doing manually
+datalad create -d . data/mriqc
+
+# Sample run without any parallelization, and doing both levels (participant and group)
+reproman run --follow -r "${RM_RESOURCE}" --sub "${RM_SUB}" --orc "${RM_ORC}" \
+		 --jp container=containers/bids-mriqc data/bids data/mriqc participant,group
```

Comment:
so the original, datalad command would be
```
datalad containers-run -n containers/bids-mriqc data/bids data/mriqc participant,group
```
TODO : add inputs/outputs specification